src/components/CreateVmWizard/steps/Storage.js

```diff
@@ -210,11 +210,12 @@ class Storage extends React.Component {
                 title='Bootable flag'
               />
             }
-            <FieldLevelHelp
-              content={this.bootableInfo(rowData.bootable)}
-              inline
-              className={style['disk-size-edit-label']}
-            />
+            <div className={style['bootable-info-tooltip']}>
```

Comment:
I'd rather see this on the `className` on `InfoToolTip` itself.  That would require `InfoToolTip` to accept an extra set of classNames it will append to its internal list.  But it would make this bit of code look a lot better.